spec/css/selector.hrx

```diff
@@ -55,6 +55,85 @@
   c: d;
 }
 
+<===>
+================================================================================
+<===> attribute/quoted_non_identifier/input.scss
+// Quotes should be preserved when the string they contain is not an identifier.
+// See https://github.com/sass/dart-sass/issues/598.
+[a="b."] {c: d}
+
+<===> attribute/quoted_non_identifier/output.css
+[a="b."] {
+  c: d;
+}
+
+<===>
+================================================================================
+<===> placeholder/pseudoselectors/matches/solo/options.yml
+---
+:todo:
+- sass/libsass#2422
+
+<===> placeholder/pseudoselectors/matches/solo/input.scss
+// Style rules with placeholders should be deleted if they can't match any real
+// elements.
```

Comment:
Rather than repeating this across multiple tests, it's probably a good idea to either:

* Update the comment to relate to the specific behavior each test is covering (as it differs from other tests), or...

* Put the tests that need the same comment in a shared (virtual) directory so they can share a single `README.md` file with this comment.

Reply:
Done. I've added specific comments for each test.